ssl/quic/quic_port.c

```diff
@@ -686,18 +862,29 @@ static void port_default_packet_handler(QUIC_URXE *e, void *arg,
         goto undesirable;
 
     /*
-     * Try to process this as a valid attempt to initiate a connection.
-     *
+     * TODO: there should be some logic similar to accounting half-open
+     * states in TCP. If we reach certain threshold, then we want to
+     * validate clients.
+     */
+    if (hdr.token == NULL) {
+        port_send_retry(port, &e->peer, &hdr);
+        goto undesirable;
+    } else if (port_validate_token(&hdr, &odcid) == 0) {
+        goto undesirable;
+    } else {
+        port_bind_channel(port, &e->peer, &hdr.src_conn_id, &hdr.dst_conn_id,
+                          &odcid, &new_ch);
+    }
+
+    /*
      * The channel will do all the LCID registration needed, but as an
      * optimization inject this packet directly into the channel's QRX for
      * processing without going through the DEMUX again.
      */
```

Comment:
this is captain obvious speaking: the `port_on_new_conn()` becomes currently unused function. but it will come back once we will have answer for TODO note few lines above.

Reply:
You can just use `#if 0`